### PR TITLE
README.md replace broken button image

### PR DESCRIPTION
**Pull request by schwesig**
README.md
- replace broken button image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ We use Netlify to preview PR changes. Each PR will show a Netlify check that can
 
 CI will deploy to GitHub pages automatically on every push to default branch as well on daily schedule. You can trigger a new build manually if you have _write_ permissions on this repo by simply clicking **Run workflow** button on the [CI workflow details screen here](https://github.com/operate-first/operate-first.github.io/actions/workflows/build_job.yaml).
 
-![Manual CI trigger](misc/manual_ci_trigger.png)
+[![image](https://user-images.githubusercontent.com/89909507/173858794-8d51d689-036a-4397-9796-8b37d2e4634f.png)](https://github.com/operate-first/operate-first.github.io/actions/workflows/build_job.yaml)
```
Comment (durandom) [inline README.md:32]: @schwesig I don't think user-images is the right way. Just add the image as an asset to a subdirectory. E.g. `/docs`

**Review by durandom — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (schwesig) [reply README.md]: thanks, changed it, thanks for feedback.

**Comment by bryanmontalvan** [comment]
A nice touch, love it!
/lgtm

**Review by durandom — APPROVED** [review]

**Pull request merged**